automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/JupyterModel.scala

```diff
@@ -1,14 +1,43 @@
 package org.broadinstitute.dsde.workbench.leonardo
 
+case class Content() {}
+
 case class ContentItem (
                        `type`: String,
-                       mimetype: String,
+                       mimetype: Option[String],
                        writable: Boolean,
                        name: String,
                        format: String,
                        created: String,
                        content: Option[String],
                        last_modified: String,
-                       path: String,
+                       path: String ,
                        size: Int
 )
+
+//TODO: the below objects are a representation of the notebook
```

Comment:
Is this comment still true? I see below you are deserializing `NotebookContentItem` from the Jupyter API.